frontend/src/scenes/insights/views/InsightsTable/InsightsTable.tsx

```diff
@@ -237,6 +237,7 @@ export function InsightsTable({
             useURLForSorting={insightMode !== ItemMode.Edit}
             rowRibbonColor={isLegend ? (item) => getSeriesColor(item.seriesIndex, compare || false) : undefined}
             firstColumnSticky
+            maxHeaderWidth="10rem"
```

Comment:
Makes sense to me, though 10rem seems really narrow. Can we go for something luxurious like 32rem as the initial limit? Worried someone might complain we're truncating _too much_, but not sure how warranted this is

Reply:
Makes sense! I tried 32 and it was a bit wide (it still takes up most of the width of the table on my laptop screen) but increased it to 20!